pkg/datapath/linux/node_ids.go

```diff
@@ -280,6 +284,13 @@ func (n *linuxNodeHandler) RestoreNodeIDs() {
 	}
 
 	n.registerNodeIDAllocations(nodeIDs)
+	for ip := range incorrectNodeIDs {
+		if err := n.unmapNodeID(ip); err != nil {
+			log.WithError(err).WithFields(logrus.Fields{
+				logfields.IPAddr: ip,
+			}).Warn("Failed to remove a incorrect node IP to node ID mapping")
+		}
+	}
```

Comment:
Should we also print a warning about the fact we found such incorrect mappings? I understand right now we expect some users have accumulated hundreds of incorrect mappings, but longer term we won't be expecting any.

Reply:
done.